examples/graph_sim/bench/graph_bench.cpp

```diff
@@ -0,0 +1,201 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ * Copyright(c) 2021 Intel Corporation
+ */
+
+#include <gflags/gflags.h>
+#include <sys/time.h>
+
+#include <atomic>
+#include <cstdint>
+#include <random>
+#include <string>
+#include <thread>
+#include <vector>
+
+#include "graph_impl.hpp"
+#include "options.hpp"
+
+using namespace google;
+
+DEFINE_bool(construct, true, "Construct the graphdb.");
+DEFINE_bool(search_degree, false, "Use the bfs search degree algo.");
+DEFINE_bool(topn, false, "Use the topn algo.");
+DEFINE_int32(topn_num, 10, "The top num of the search with topn!");
+
+DEFINE_int64(vertex_nums, 10000000,
+             "The number of the vertexes in the simple graphdb.");
+DEFINE_int64(vertex_id_range, 30000000, "The range of the vertex's id.");
+DEFINE_int32(client_threads, 32,
+             "The number of the threads to run the graphdb's workload.");
+DEFINE_string(engine_name, "kvdk",
+              "The name of the engine, if you want to create other"
```

Comment:
so far, it supports kvdk, rocksdb and memory engine, right?

Reply:
yes